BAZProjectC1/Components/CarouselMovies/movieCarouselCell.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  movieCarouselCell.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 22/09/22.
+//
+
+import UIKit
+
+class movieCarouselCell: UICollectionViewCell {
+    static let reuseIdentifier = String(describing: movieCarouselCell.self)
+    private lazy var backView: UIView = {
+        let backView = UIView(frame: contentView.bounds)
+        backView.layer.cornerRadius = RounderBorderStyleForView.rounded.rawValue
+        let gradient = CAGradientLayer()
+        gradient.frame = backView.bounds
+        gradient.colors = [UIColor.appColorGrayPrimary.withAlphaComponent(0.1).cgColor,
+                           UIColor.appColorBlack.withAlphaComponent(0.875).cgColor]
+        gradient.startPoint = CGPoint(x: 0.5, y: 0)
+        gradient.endPoint = CGPoint(x: 0.5, y: 0.9)
+        backView.layer.insertSublayer(gradient, at: 0)
+        return backView
+    }()
+    
+    public lazy var movieTitle: UILabel = {
+        let frame = CGRect(x: 0, y: 0, width: 0, height: 0)
+        let movieTitle = UILabel(frame: frame)
+        movieTitle.translatesAutoresizingMaskIntoConstraints = false
+        movieTitle.text = "movieTitle"
+        movieTitle.numberOfLines = 0
+        movieTitle.adjustsFontSizeToFitWidth = true
+        movieTitle.sizeToFit()
+        movieTitle.textColor = UIColor.appColorWhitePrimary
+        movieTitle.textAlignment = .left
+        return movieTitle
+    }()
+    
+    public lazy var movieRanking: UILabel = {
+        let frame = CGRect(x: 0, y: 0, width: 0, height: 0)
+        let movieRanking = UILabel(frame: frame)
```

Comment:
Si vas a usar constrains cualquier configuración de frames será ignorada. Considera usar .zero como parámetro del frame.

```suggestion
        let movieRanking = UILabel(frame: .zero)
```